notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>Also need to clean up the training job, the service account and the TensorBoard instance if created in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>done. For the custom job, I did not see a "delete" option in &nbsp;"gcloud ai custom-jobs", thus I give the users a link to follow so that they can delete in UI.</p>